packages/files-ui/cypress/tests/file-management-spec.ts

```diff
@@ -60,7 +60,7 @@ describe("File management", () => {
           .should("be.visible")
           .click()
         homePage.moveSelectedButton().click()
-        moveItemModal.folderList().contains("Testing").click()
+        moveItemModal.folderList().contains(`${folderName}`).click()
```

Comment:
Should work like this.
```suggestion
        moveItemModal.folderList().contains(folderName).click()
```

Reply:
This one was forgotten